### PR TITLE
Add endpoint to get a PE number

### PR DESCRIPTION
**Pull request by patricksmithdds**
This PR supports verifying PE numbers in atst:

* Adds a new model for PE numbers
* Adds a python script to download the list of PE numbers from a Rackspace CDN and upserts them into the database
* Adds an API endpoint that allows retrieving a PE number
* Adds some docs for the API endpoints

### CODE REVIEW

**Comment by briandds** [comment]
I ran `script/update` on this branch and got this output:
```
 Running upgrade 464c56bfb2d3 -> aae012d5f07e, Add PENumber model
Traceback (most recent call last):
  File "script/ingest_pe_numbers.py", line 7, in <module>
    from fundz.make_app import make_app, make_config
ModuleNotFoundError: No module named 'fundz'
```

**Comment by patricksmithdds** [comment]
@briandds It worked for me, so I suspect it has something to do with the python path? I added a commit to include the project root in the path -- can you try again and see if that fixes it?

File: script/ingest_pe_numbers.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    )
+    db.session.execute(do_update)
+    db.session.commit()
+
+
+if __name__ == "__main__":
+    config = make_config()
+    app = make_app(config)
+    db = SQLAlchemy(app)
+    url = "http://c95e1ebb198426ee57b8-174bb05a294821bedbf46b6384fe9b1f.r31.cf5.rackcdn.com/penumbers.csv"
```
Comment (richard-dds) [inline script/ingest_pe_numbers.py]: Might be a good idea to put this URL into the app config.

**Comment by briandds** [comment]
@patricksmithdds -- Yep, that fixed it!

File: script/ingest_pe_numbers.py (hunk 1 above)
Comment (briandds) [reply script/ingest_pe_numbers.py]: +1

Comment (patricksmithdds) [reply script/ingest_pe_numbers.py]: Good call -- did that in 9237261.

**Review by briandds — APPROVED** [review]

**Review by richard-dds — APPROVED** [review]

**Pull request merged**